.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "node"
+  - "lts/10"
+  - "lts/8"
+cache: npm
+os:
+  - osx
+  - precise
```

Comment:
`- windows` can be added as well :-)

`xenious` is another available Ubuntu distro, depending of what kind of environment you are looking for.

Aside of that 👍

Reply:
Appreciate that feedback – will add both of those 🙏